server/plugins/Plugin.ts

```diff
@@ -1,15 +1,53 @@
-export interface PluginReturn {
-	error: boolean;
-	key: string;
-	message: string;
-}
-
-export interface Plugin<T> { // Separate schema and notifier
-	init(): Promise<Notifier<T>>;
-	schema(): string;
-}
-
-export interface Notifier<T> {
-	sendMessage(message: string, config: T): Promise<PluginReturn[]>;
-	check(config: any): Promise<T>;
-}
+export interface PluginReturn {
+	error: boolean;
+	key: string;
+	message: string;
+}
+
+export interface Plugin<T> { // Separate schema and notifier
+	init(): Promise<Notifier<T>>;
+	schema(): string;
+}
+
+export interface Notifier<T> {
+	sendMessage(message: string, config: T): Promise<PluginReturn[]>;
+	check(config: any): Promise<T>;
+}
+
+export interface PluginMasterReturn {
+	live_site: LiveSiteConfigType;
+	slack: SlackConfigType;
+	twilio: TwilioConfigType;
+	twitter: TwitterConfigType;
+}
+
+export interface LiveSiteConfigType {

```

Comment:
There are configs defined in each plugin, use those. See if you can do any trick with reading os dir or at least add a comment in PluginMasterReturn that each new plugin needs to be registered there.